Installer_Linux.sh

```diff
@@ -78,6 +78,14 @@ if [ ! -d /data/hdd/misc ]; then
     sudo chown -R ${USER:=$(/usr/bin/id -run)}:$USER /data/hdd/misc
 fi
 
+echo
+echo "Acquiring shape files..."
+if [ ! -d /data/hdd/misc/shapes]; then
```

Comment:
Bash syntax is a little finicky here. I believe you need to put a space between the right square bracket and the "s" character to avoid a syntax error:

```sh
if [ ! -d /data/hdd/misc/shapes ]; then
```